### PR TITLE
🌐 Add Russian translation for `docs/tutorial/path-operation-configuration.md`

### PR DESCRIPTION
**Pull request by TabarakoAkula**

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 02497fd16439aaecd566388f0e48b3f83f0e6711 at: https://648dfde100fa3a32580d503e--fastapi.netlify.app

File: docs/ru/docs/tutorial/path-operation-configuration.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    ```Python hl_lines="3  17"
+    {!> ../../../docs_src/path_operation_configuration/tutorial001.py!}
+    ```
+
+Этот код состояния будет использован в ответе, и будет добавлен в схему OpenAPI.
+
+!!! note "Технические детали"
+    Вы также можете использовать `from starlette import status`.
+
+    **FastAPI** предоставляет тот же `starlette.status`, что и `fastapi.status` для удобства разработчкиа. Но идет он непросредственно от Starlette.
```
Comment (Alexandrhub) [inline docs/ru/docs/tutorial/path-operation-configuration.md:39]:
```suggestion
    **FastAPI** предоставляет тот же `starlette.status`, что и `fastapi.status` для удобства разработчика. Но идет он непосредственно от Starlette.
```

File: docs/ru/docs/tutorial/path-operation-configuration.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+Они будут добавлены в схему OpenAPI и будут использованы в автоматической документации интерфейса:
+
+<img src="/img/tutorial/path-operation-configuration/image01.png">
+
+### Теги с перечислениями
+
+Если у вас большое приложение, под конец вы можете набрать **несколько тегов**, и вы можете хотеть убедиться в том, что всегда используете **один и тот же тег** для связанных *операций пути*.
+
+В этих случаях, имеет смысл хранить теги в `Enum`.
```
Comment (Alexandrhub) [inline docs/ru/docs/tutorial/path-operation-configuration.md:71]:
```suggestion
В этих случаях, имеет смысл хранить теги в классе `Enum`.
```

File: docs/ru/docs/tutorial/path-operation-configuration.md [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+**FastAPI** поддерживает это также, как и с обычными строками:
+
+```Python hl_lines="1  8-10  13  18"
+{!../../../docs_src/path_operation_configuration/tutorial002b.py!}
+```
+
+## Краткое и развёрнутое содержание
+
+Вы можете добавить `summary` и `description`:
```
Comment (Alexandrhub) [inline docs/ru/docs/tutorial/path-operation-configuration.md:81]:
```suggestion
Вы можете добавить параметры `summary` и `description`:
```

File: docs/ru/docs/tutorial/path-operation-configuration.md [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+    ```
+
+=== "Python 3.6+"
+
+    ```Python hl_lines="21"
+    {!> ../../../docs_src/path_operation_configuration/tutorial005.py!}
+    ```
+
+!!! info "Дополнительная информация"
+    Помните, что `response_description` относится конкретно к ответу, а `description` в основном относится к *операциям пути*.
```
Comment (Alexandrhub) [inline docs/ru/docs/tutorial/path-operation-configuration.md:152]:
```suggestion
    Помните, что параметр `response_description` относится конкретно к ответу, а `description` в основном относится к *операциям пути*.
```

**Review by Alexandrhub — APPROVED** [review]

File: docs/ru/docs/tutorial/path-operation-configuration.md [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+
+```Python hl_lines="16"
+{!../../../docs_src/path_operation_configuration/tutorial006.py!}
+```
+
+Он будет четко помечен как устаревший в интерактивной документации:
+
+<img src="/img/tutorial/path-operation-configuration/image04.png">
+
+Проверьте, как устаревшие и не-устаревшие *операции пути* будут выглядеть:
```
Comment (ivan-abc) [inline docs/ru/docs/tutorial/path-operation-configuration.md:173]:
```suggestion
Проверьте, как будут выглядеть устаревшие и не устаревшие *операции пути*:
```

**Review by ivan-abc — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 13d2845152eab38d2e543506b0ca8bdc351fd209 at: https://64917538e1c1e01089465b67--fastapi.netlify.app

**Pull request merged**